crates/wasmtime/src/runtime/fiber.rs

```diff
@@ -354,6 +354,7 @@ impl<T> StoreContextMut<'_, T> {
     /// # Panics
     ///
     /// Panics if this is invoked outside the context of a fiber.
+    #[allow(dead_code)]
```

Comment:
I made this change to fix `cargo check -p wasmtime --no-default-features --features async,std` (previously `cargo check -p wasmtime --no-default-features --features async`) but I'm not sure if that's an expected behavior change.